core/src/utils.jl

```diff
@@ -110,12 +110,14 @@ end
 """Compute the storages of the basins based on the water level of the basins."""
 function get_storages_from_levels(
     basin::Basin,
-    levels::Vector{Float64},
+    levels::Vector,
 )::Tuple{Vector{Float64}, Bool}
-    storages = Float64[]
+    storage = get_storage_from_level(basin, 1, levels[1])
+    storages = [storage]
 
-    for (i, level) in enumerate(levels)
-        push!(storages, get_storage_from_level(basin, i, level))
+    for (i, level) in enumerate(levels[2:end])
+        storage = get_storage_from_level(basin, i + 1, level)
+        push!(storages, storage)
```

Comment:
Is there a reason for this change? It looks less readable to me.

Reply:
Turns out I don't need this change (: